### PR TITLE
docs(core): [I18N help wanted!] Update TSBS benchmark and add explainer

### PR DESCRIPTION
**Pull request by oliver-daniel**
Replaces the previous benchmark comparing QuestDB to its competitors over increasing number of workers, to one comparing ingest speeds with fixed compute and increasing cardinality.

**IMPORTANT**: as the PR stands, only the English-language README has been updated. Help would be sincerely appreciated to bring this improvement to our READMEs in:

- [x] `zh-cn` (Mandarin) (thank you @1998code!)
- [x] `zh-hk` (Cantonese) (thank you @terasum!)
- [ ] `ar-dz` (Arabic)
- [x] `it-it` (Italian) (thank you @TheZalRevolt!)
- [ ] `ua-ua` (Ukrainian)
- [ ] `es-es` (Spanish)
- [ ] `pt` (Portuguese)
- [ ] `ja-ja` (Japanese)
- [x] `tr-tr` (Turkish) (thank you @kerimsenturk5734!)
- [x] `hn-in` (Hindi) (thank you @heysagnik!)

## NOTE FOR TRANSLATORS
Part of this PR introduces a new image, `.github/questdb7.3.10-tsbs-benchmark.png`, which is **NOT** available on the `master` branch. Please base your PRs off of  _this branch_ (`doc/how-questdb-compares`) and ensure that all images appear properly before submitting. If you've already created your PRs, try rebasing your changes off of this branch and ensure that your PR target branch is correct. Thank you, and apologies for previous lack of clarity!

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 34d9a16fbdd1ede0a70c43502f68cb44333c1216
</p>

File: README.md [hunk 1]
```diff
@@ -126,13 +126,15 @@ compares QuestDB to other open source time series databases spanning functionali
...
+results using the `cpu-only` use case with 12 workers on 48 CPUs and 96GB RAM:
 
 <div align="center">
-    <img alt="A chart comparing the ingestion rate of QuestDB, InfluxDB and TimescaleDB." src=".github/readme-benchmark.png" width="600"/>
+    <img alt="A chart comparing the ingestion rate of QuestDB, InfluxDB and TimescaleDB." src=".github/questdb7.3.10-tsbs-benchmark.png" width="600"/>
   </a>
 </div>
 
+Competing TSDBs like InfluxDB and Timescale take a performance hit as the number of hosts grows. By contrast, not only does QuestDB soar above the competition at any host count, it maintains its high ingestion speed as cardinality increases.
+
```
Comment (tris0laris) [inline README.md:137]: I would flip this around:

This benchmark measures the ingestion rate (rows/second) for different numbers of hosts 100, 1K, 100K and 10M hosts. The larger the number of hosts, the higher the dataset cardinality. As the number of hosts increases, QuestDB maintains its ingestion speed above 2.4M rows/s. By contrast, as the dataset cardinality increases, InfluxDB and TimescaleDB take a performance hit, with the ingestion rate degrading significantly.

Comment (oliver-daniel) [reply README.md:137]: Nice one.

**Review by goodroot — APPROVED** [review]
LGTM!

**Comment by oliver-daniel** [comment]
👋 Pinging a handful of folks who were major contributors to previous i18n efforts!

Just wanted to bring these changes to your attention, and if you find yourself with the time and energy, additions to this PR (or separate PRs) to bring the READMEs back in sync would be sincerely appreciated. Your work helps make QuestDB accessible to linguistic communities worldwide!

@HouariZegai
@heysagnik 
@TheZal 
@4N1Z 
@Jovin247 
@kerimsenturk5734 
@bsmth @dragonlee8 @pswu11 @terasum
@1998code

**Review by mtopolnik — APPROVED** [review]

**Comment by TheZalRevolt** [comment]
I'll be happy to help for the italian part but unfortunately I will be on holiday without the pc untill the 3rd of april, is that ok?

**Comment by kerimsenturk5734** [comment]
I glad to update for Turkish part. I will be updating as soon as possible.

**Comment by 1998code** [comment]
```zh-HK``` updated 🙏

**Comment by oliver-daniel** [comment]
Hi friends, waking up to so many enthusiastic replies warms my language-loving heart. <3

Please be advised that I could have been a bit more clear about how to go about submitting PRs for your contribution -- TLDR it's important to base your work on the branch of this PR, `doc/how-questdb-compares`, rather than `master` in order to include a new image that is not yet available on the main branch. That being said, once _this_ PR is merged, the image should be available and the link will work in your PR. So, don't fret too much if you started off `master`, as this English PR will likely be merged first.

Thank you, grazie, 谢谢, 謝謝, etc.!

**Comment by 1998code** [comment]
```zh-hk``` translation => (https://github.com/questdb/questdb/pull/4332) fixed branch and images.

P.S. I cannot see the ```doc/how-questdb-compares``` branch in my old fork, even though I synced it many times (no matter in the website / GitHub desktop).  After all the tries, I deleted the repo and fork again, and then it can show again 🙈

**Comment by heysagnik** [comment]
updated `hn-in`

**Comment by kerimsenturk5734** [comment]
Hi. I am going to fork the project from ``doc/how-questdb-compares`` branch, update the Turkish readme file (for me) and create a PR by mentioning this PR. Is that what should i do? I ask that to be sure. Please correct me if i am wrong.

**Comment by oliver-daniel** [comment]
Hey @kerimsenturk5734, sounds perfect. Thank you!

**Comment by TheZalRevolt** [comment]
Done! :)

**Pull request closed without merging**